### DISCUSSION TITLE
Unable to use GPU on FSL

### DISCUSSION THREAD

**Discussion by elai125** [neuropoly/axondeepseg#577] [Help needed] [answered]
Hi,

I'm a newcomer to ADS and it's an awesome software. I followed advice from issues #469 and #490, but still encountered a different issue.

![image](https://user-images.githubusercontent.com/93625629/139965561-129d30a5-0f34-4261-9f22-7d518814ae62.png)

I was able to open and use Fsleyes, and I installed tensorflow-gpu 1.13.1 in the virtual environment. I also installed CUDA 11 in my Ubuntu 18.04. However, it seems that my GTX GeForce 1060 was not recognized, and my CPU was used to perform segmentation instead. Is there a way to make ADS use my GPU?

Thanks,
elai125

**Comment by mathieuboudreau** [comment, marked as answer]
Hi @elai125,

It seems that your issue isn't how to use ADS using GPU, but more fundamentally how to setup your computer to use Tensorflow's GPU feature. For this, I'd recommend you test and develop your computer's CUDA/Tensorflow setup it on Tensorflow directly first, before trying on ADS, maybe using one of their examples.

https://www.tensorflow.org/guide/gpu
